mailpoet/lib/AdminPages/Pages/DynamicSegments.php

```diff
@@ -186,6 +186,7 @@ private function getNewslettersList(): array {
       $result[] = [
         'id' => (string)$newsletter->getId(),
         'subject' => $newsletter->getSubject(),
+        'name' => $newsletter->getPresentableName(),
```

Comment:
NewsletterResponseBuilder uses `campaign_name` and here is `name`. Is there a reason for that? I think we should unify the used name for all places. It can be confusing to mix those two.